DQMOffline/Lumi/plugins/ZCounting.cc

```diff
@@ -609,8 +609,6 @@ void ZCounting::analyzeElectrons(const edm::Event& iEvent, const edm::EventSetup
   }
 
   // Good vertex requirement
```

Comment:
Now this comment hangs uselessly... I would also remove it, together with the two other lines
Once you do so, could you please squash all commits into one?

Reply:
The outdated comment is still there: could you please remove it (and squash all commits after it)?

Reply:
I would like to maintain the vertex condition for the electrons, since it was supposed to be removed only for the muons (analyzeMuons).